apps/crn-server/src/routes/research-outputs.route.ts

```diff
@@ -48,14 +49,25 @@ export const researchOutputRouteFactory = (
   researchOutputRoutes.post('/research-outputs', async (req, res) => {
     const { body, loggedInUser } = req;
 
+    // Middleware makes sure that the user is logged in at this stage
+    /* istanbul ignore next */
```

Comment:
I think in such a case maybe its better to cast `req` as req with the loggedInUser? in fact you don't need to 'cast', just give the req to the `post` method which accepts a generic